src/deploy.go

```diff
@@ -233,18 +269,28 @@ func addFiles(form uint8, parent *html.Node, files []string) {
 	}
 }
 
+/*
+* Returns if the host is blank
```

Comment:
I'll stop commenting on the comments, but please remove any comment which doesn't add information.  In general comments should explain 'why' or give context.  If you need a comment to reiterate what the function does, you need shorter functions, not more comments.